src/app/model/staypoints.ts

```diff
@@ -39,3 +46,36 @@ export function writeStaypointsToGeoJSON(sp: StayPoints, path: string) {
     }
   })
 }
+
+export function writeStaypointClusterArrayToGeoJSON(
+  spcs: StayPointCluster[],
```

Comment:
You could write this out as `stayPointClusters` instead of `spcs`.

Reply:
Agreed, I changed it.